docs/MapColonies/Raster/services/PolygonParts/README.md

```diff
@@ -1806,3 +1806,44 @@ We'll invoke a request with the following body:
 ```
 
 </details>
+
+### ResultType
+
+By default, `resultType` is set to `results`. If a user wants to get the number of parts in a feature, `resultType=hits` should be used.
+
+- **`numberMatched`**: Indicates how many hits(parts) there are.
+- **`next`**: Provides the link to fetch the response itself.
```

Comment:
Please add a warning that the link does not include pagination parameters and will return all results for the layer. This should be avoided and either manualy add the `count` parameter or make a seperate request with pagination.